src/js/components/Organization.js

```diff
@@ -113,6 +113,13 @@ export default class Organization extends Component {
 
   render () {
     const person = this.props.person;
+    let givenName = person.givenName;
+    // check to see if givenName includes single letter (middle initial)
+    // after first name, and trim it
+    if (givenName && givenName.indexOf(' ') === givenName.length - 2) {
```

Comment:
I believe this expression does the trick:

`\w+\s(\w\.?\s)\w+`

It already accounts for the optional `.` in the name.